### DISCUSSION TITLE
Race Chrono iOS WiFi Connection - Working

### DISCUSSION THREAD

**Discussion by Ralphy140** [renatobo/bonogps#33] [Show and tell]
Hi All, 

I have just tested with Race Chrono on iOS over the wifi connection and it is working great out of the box. no need to make any config changes to the GPS and just follow the setup guide below

![image](https://user-images.githubusercontent.com/85327073/120946167-ecf1ac00-c77e-11eb-8136-b779deee6011.png)
 
![image](https://user-images.githubusercontent.com/85327073/120946178-f67b1400-c77e-11eb-8bd5-d144460ea8c5.png)
![image](https://user-images.githubusercontent.com/85327073/120946186-00047c00-c77f-11eb-8e1a-1e7751897b70.png)
![image](https://user-images.githubusercontent.com/85327073/120946194-07c42080-c77f-11eb-8fb6-0f23db3696b9.png)
![image](https://user-images.githubusercontent.com/85327073/120946205-10b4f200-c77f-11eb-8b64-5b9250b6a74e.png)
![image](https://user-images.githubusercontent.com/85327073/120946230-20343b00-c77f-11eb-96b7-6807909b4b73.png)

Enjoy, I have also been working on the BLE stack and have had scucess in that race chrono is able to identify and detect the device using BLE but i have not been able to send the correct message format yet as it takes a bit more work. you would also need to modify the BLE UUID settings to be changed based on the app that you are conneting to based on the attached projects.

**Comment by renatobo** [comment]
Thank you @Ralphy140  for testing this!
Would you like to update the documentation with a pull-request?

**Reply by Ralphy140** [reply]
done and done

**Comment by soczkers** [comment]
tried that solution. Connected well, gps singal is avaliable in racechrono however : when I was on track, and blocked the screen(motorcycle riding) it look like IOS somehow disconnected wifi. I will try to debug this more maybe next time, but did you observed similar issue @Ralphy140 ? I'm not super familiar with IOS - that was my friends mobile - maybe there is some save-energy option which is enabled

**Reply by Ralphy140** [reply]
Not had this problem, however i have always had the screen turned on for Race Chrono as i use it as a dashboard in the car. Some apps in iOS do not play nice with the memory/power save features when the screen is off. I know that Track Addict specifically calls out that it does not support background activities(when the app is running but screen is off) I found that one out the hard way :(
